extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -291,6 +295,11 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     }
     this.ampAnalyticsConfig_ = null;
   }
+
+  /** @override */
+  getPreconnectUrls() {
+    return 'https://googleads.g.doubleclick.net';
```

Comment:
was there any reason specifically you want this to be an array instead of a string? the way it was previously implemented handled it either as a string or an array of strings.